back/app/models/idea.rb

```diff
@@ -155,6 +155,11 @@ class Idea < ApplicationRecord
 
   scope :order_trending, -> { TrendingIdeaService.new.sort_trending(where('TRUE')) }
 
+  scope :from_survey_project, -> { joins(:project).where(projects: { participation_method: 'native_survey' }) }
+  scope :from_survey_phase, -> { joins(:creation_phase).where(phases: { participation_method: 'native_survey' }) }
+  scope :from_survey, -> { where(id: from_survey_project.select(:id)).or(where(id: from_survey_phase.select(:id))) }
+  scope :not_from_survey, -> { where.not(id: from_survey.select(:id)) }
```

Comment:
New scopes. First 3 used to make 4th possible.

`not_from_survey` scope used in `TrendingIdeaService`.

Reply:
Note: I also [checked with methods squad](https://citizenlabco.slack.com/archives/C031GG2M8BH/p1681401966511179), and it seems we do not use Ideas in any other 'non-standard' way - just for native Surveys.

Reply:
Scopes removed in favor of Sebi's excellent 'idea' to use `IdeaFinder`.